docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,204 @@
+
+# Setting up Service Catalog for API Aggregation in Kubernetes
+
+The aggregator is a server that sits in front of everything. 
+
+We need to provide a set of certificates to be provided as a
+certificate bundle to the APIService apiregistration endpoint.
+
+For development purposes, it is convenient to use the existing CA
+automatically set up by the kubernetes development environment. The
+[script in contrib](../contrib/svc-cat-apiserver-aggregation-tls-setup.sh)
+generates a fresh CA and certificate setup, without using any existing
+kubernetes infrastructure CAs or certificates. This script should be
+`source`ed to define all of the variables it contains in the current
+shell process.
+
+The aggregator is a new feature of kubernetes and is an alpha API
```

Comment:
missing `server` after `API`.  I'd move this paragraph up to the top where brad made his comments.